modules/administration/pages/monitoring.adoc

```diff
@@ -4,6 +4,13 @@
 You can monitor your {productname} environment using Prometheus and Grafana.
 {productname} Server and Proxy are able to provide self-health metrics, or install and manage a limited number of Prometheus exporters on Salt clients.
 
+[NOTE]
+====
+Prometheus fetches metrics using a pull mechanism, so it must be able to establish network communications with monitored clients.
+If your clients are using a non-routable network with a reserved IP range, they will not be able to be contact the Prometheus server.
```

Comment:
This is not correct. The clients never contact the Prometheus server. It is the Prometheus server that contacts the clients, and that is were the problem is. Also, the reserved IP range is not meaningful here, and the definition of non-routable network can be confusing. I would remove this sentence altogether.

Reply:
@cavalheiro Removed the problem sentence.